sql/setup_main.sql

```diff
@@ -32,6 +32,17 @@ BEGIN
     AFTER INSERT OR UPDATE OR DELETE ON _timescaledb_catalog.hypertable_index
     FOR EACH ROW EXECUTE PROCEDURE _timescaledb_internal.on_change_hypertable_index();
 
+    DROP TRIGGER IF EXISTS trigger_main_on_change_hypertable_trigger
```

Comment:
Could you remind we why we need `DROP TRIGGER IF EXISTS` on some but not all triggers ?